fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/MessageValidator.kt

```diff
@@ -231,6 +242,27 @@ object MessageValidator {
       return this
     }
 
+    /**
+     * Helper method to check that a message has a valid structure.
+     *
+     * @param message the message to check
+     */
+    fun validMessage(message: Any): MessageValidatorBuilder {
+      message as Map<*, *>
+      isNotEmptyBase64(message["data"] as String?, "data")
+      isNotEmptyBase64(message["sender"] as String?, "sender")
+      isNotEmptyBase64(message["signature"] as String?, "signature")
+      isNotEmptyBase64(message["message_id"] as String?, "message_id")
```

Comment:
I see a possible optimization to the code here: what about extending the `isNotEmptyBase64` method to support as input a variable number of messages? It would reduce a bit the code repetition here and in the future

Reply:
makes sense!